packages/block-library/src/post-tags/edit.js

```diff
@@ -1,42 +1,77 @@
 /**
  * WordPress dependencies
  */
-import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { useEntityProp } from '@wordpress/core-data';
+import { Warning, __experimentalBlock as Block } from '@wordpress/block-editor';
 import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
-function PostTagsDisplay() {
-	const [ tags ] = useEntityProp( 'postType', 'post', 'tags' );
+export default function PostTagsEdit( { context } ) {
+	const [ tags ] = useEntityProp(
+		'postType',
+		context.postType,
+		'tags',
+		context.postId
+	);
 	const tagLinks = useSelect(
 		( select ) => {
 			const { getEntityRecord } = select( 'core' );
 			let loaded = true;
-			const links = tags.map( ( tagId ) => {
-				const tag = getEntityRecord( 'taxonomy', 'post_tag', tagId );
-				if ( ! tag ) {
-					return ( loaded = false );
-				}
-				return (
-					<a key={ tagId } href={ tag.link }>
-						{ tag.name }
-					</a>
-				);
-			} );
+			let links;
```

Comment:
I've got an infamous adversion to `let`, with the assumption that usually using `const` can help simplify the code. 😄 

In this case `links` only depends on the existence of `tags` which, if undefined, would throw an error on `map`. It can be solved in several ways.
I often like using Lodash, whose `map` is guarded against undefined:
```js
import { map } from 'lodash';
const links = map( tags, tagId => { /* ... */ } );
// when tags is undefined, returns an empty array
```

Otherwise, we can use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) `?.`:
```js
const links = tags?.map( tagId => { /* ... */ } );
// when tags is undefined, returns undefined
```

Either way, we'd be removing a `let` and an `if`. 🙂

Reply:
@Copons Great suggestion! I'll keep this in my back pocket for next time 🙂. I went in the direction of optional chaining. All things being relatively equal, I tend to prefer native javascript over third-party libraries.